### DISCUSSION TITLE
Server is not running with k8s

### DISCUSSION THREAD

**Discussion by AEncinas19** [srl-labs/learn-srlinux#68] [Q&A] [answered]
Hello everyone.
I'm deploying with k8s the Nokia SR Linux 22.11.1 in a container allocated in a pod.
When I execute the command: 
microk8s kubectl -n <namespace> exec -it <pod> -c router sr_cli I get the following error:
```
12:27:38.561` [WARNING] (MainThread) Failed to contact the AAA server. Further communication will be blackholed until connection succeeds
Using configuration file(s): ['/etc/opt/srlinux/srlinux.rc']
Welcome to the srlinux CLI.
Type 'help' (and press <ENTER>) if you need any help using this.
Warning: connection with server is down
--{ running }--[  ]--
```
and if I execute de command: enter candidate, I get:
`
Error: Server is not running
--{ running }--[  ]--
`
In the documentation I haven't seen anything to configure the AAA server. 
What could I do?

**Comment by hellt** [comment, marked as answer]
Hi,
SR Linux Pod has to have a certain cmd/entrypoint to run, amongst other things.
You can check our k8s controller that schedules srlinux pods under KNE -- https://github.com/srl-labs/srl-controller

**Comment by AEncinas19** [comment]
Thank you very much Hellt. I think I would like to do something different. I have a network already deployed and I want to substitute a Linux router with a SR Linux, but this router should be allocated in a pod. 
I think that KNE builds a topology, but I only need to install the SR Linux.

Thank you for the support.
